packages/kamino-sdk/tests/stratCreation.test.ts

```diff
@@ -1259,7 +1259,7 @@ describe('Kamino strategy creation SDK Tests', () => {
     console.log('shareData', shareData.length);
   });
 
-  it('amounts distribution to be deposited with price range Orca', async () => {
+  it.skip('amounts distribution to be deposited with price range Orca', async () => {
```

Comment:
why do we skip tests?

Reply:
the tests in this file run on mainnet as they call thje raydium/orca APIs and they use our connection so if we use the default RPC for Solana they will be very flaky and we don't want to expose the private RPCs